tests/unit/core/utils/test_compatibility.js

```diff
@@ -610,4 +614,36 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('isQuantumCompatible', () => {
+    it('returns `true` when add-on is compatible', () => {
+      const addon = createInternalAddon(createFakeAddon({
+        name: 'Firefox Multi-Account Containers',
+        compatibility: {
+          [CLIENT_APP_FIREFOX]: {
+            max: '*',
```

Comment:
You should set this to `57.0` to make it realistic

Reply:
It is realistic since it is the prod data..

Reply:
See: https://addons.mozilla.org/api/v3/addons/addon/multi-account-containers/

Reply:
There is a new test with `57.0` as `max`.

Reply:
Oh, cool. I didn't notice that prod uses `*`. I was looking at dev.